src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -236,15 +223,47 @@ public Base(final Request req) throws IOException {
                     break;
                 }
                 this.buffer.position(this.buffer.position() + 1);
-                requests.add(this.make(boundary));
+                requests.add(this.make(boundary, body));
             }
             return RqMultipart.Base.asMap(requests);
         }
         /**
+         * Init a map of params and values.
+         * @param req Request
+         * @return ConcurrentMap, emtpy if something wrong
+         * @throws IOException if fails
+         */
+        private ConcurrentMap<String, List<Request>> initMap(final Request req)
+            throws IOException {
+            final InputStream stream = new RqLengthAware(req).body();
+            this.initBuffer(stream);
```

Comment:
@2686747 this is a very bad practice, see http://www.yegor256.com/2015/12/08/temporal-coupling-between-method-calls.html